helm/cloud-provider-vsphere/charts/cloud-provider-for-vsphere/templates/podsecuritypolicy.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.podSecurityPolicy.enabled }}
```

Comment:
It was false by default. Interesting. How was it working?

Reply:
The PSPs just weren't installed -_-